rust/actyx/ax-core/src/private_key.rs

```diff
@@ -157,54 +176,6 @@ pub fn load_dev_cert(path: Option<PathBuf>) -> ActyxOSResult<DeveloperCertificat
     serde_json::from_str(&s).ax_err_ctx(ActyxOSCode::ERR_INVALID_INPUT, "reading developer certificate")
 }
 
-#[derive(Debug, Clone)]
-/// Newtype wrapper around a path to key material, to be used with
-/// structopt/clap.
-pub struct KeyPathWrapper(PathBuf);
-
-impl FromStr for KeyPathWrapper {
-    type Err = ActyxOSError;
-    fn from_str(s: &str) -> Result<Self, Self::Err> {
-        Ok(Self(s.into()))
-    }
-}
-
-impl fmt::Display for KeyPathWrapper {
-    fn fmt(&self, f: &mut fmt::Formatter<'_>) -> fmt::Result {
-        write!(f, "{}", self.0.display())
-    }
-}
-
-// NOTE(duarte): there has to be a better way of doing this
-impl TryFrom<&Option<KeyPathWrapper>> for AxPrivateKey {
-    type Error = ActyxOSError;
-    fn try_from(k: &Option<KeyPathWrapper>) -> Result<Self, Self::Error> {
-        if let Some(path) = k {
-            path.0
-                .to_str()
-                .and_then(|s| s.parse::<AxPrivateKey>().ok())
```

Comment:
I think you removed this functionality: I made it so that the key can be passed verbatim on the command line (or in an env var), avoiding the need to store it in a file

Reply:
My bad, I did not realize there is this functionality when making the issue. So now I got it why it is named `KeyPathWrapper`

Reply:
Oh I see now. It's super non obvious because the type of the wrapper is a `PathBuf`.
If we are to support both, I prefer having a flag for the inline key and another for the path. The previous way is too overloaded, both in usage and in code.

Reply:
well, it is somewhat established to read secrets from env vars, though

Reply:
I can re-add the feature with `String` instead of `PathBuf` as the main type. We can then try to parse into an `AxPrivateKey` and if it fails, read it as a path.

Code-wise this would make much more sense, client wise, it's the same